examples/fastify-ssr/src/server/handlers/app-shell.js

```diff
@@ -1,6 +1,6 @@
 const React = require('react')
 const ReactDOMServer = require('react-dom/server')
-const { ServerLocation } = require('@reach/router')
+const { StaticRouter } = require('react-router')
```

Comment:
@simoneb react-router is used here, but I don't know if it's effectively needed.

Reply:
I don't know, can you please check the docs? https://reactrouter.com/docs/en/v6/guides/ssr

Reply:
Thanks, good advice. It's not needed indeed, I'll push the changes.